### ISSUE TITLE
Fails to work if no port is specified in the URL and tika is running behind a reverse proxy

### ISSUE THREAD

**Issue by BenNewsome** [chrismattmann/tika-python#201]
Using curl I can get the contents of a file by using the domain name with a reverse proxy setup and not specifying a port.
tika-python appears to crash if no port is specified.

```
import tika
from tika import parser
parsed = parser.from_file("example.pdf, "http://tika_hostname/tika")
print(parsed["content"])
```

Expected:
  The result.
Actual:
```Traceback (most recent call last):
  parsed = parser.from_file(filename, tika_address)
  File "~/anaconda3/lib/python3.6/site-packages/tika-1.19-py3.6.egg/tika/parser.py", line 36, in from_file
  File "~/anaconda3/lib/python3.6/site-packages/tika-1.19-py3.6.egg/tika/tika.py", line 328, in parse1
  File "~/anaconda3/lib/python3.6/site-packages/tika-1.19-py3.6.egg/tika/tika.py", line 537, in callServer
```

**Comment by lamb003** [comment]
Hi Chris,
I came across this issue too. I added a check for a None port and then assumed the port from the http scheme. 443 for https and 80 for http. 
Should the PR go to the master branch?

**Comment by chrismattmann** [comment]
thanks @lamb003 yes please send against master branch. Thanks!

**Comment by lamb003** [comment]
Awesome, made a PR @chrismattmann. I can't seem to select any reviewers though.

**Comment by chrismattmann** [comment]
Fixed in #204
